### PR TITLE
Authentication option 1 take i

### PR DESCRIPTION
**Pull request by mcbride-p**
@ewill00 , @lmsurpre following on from our call last week, readme for option #1 here which you can follow at your convenience -> https://github.com/Alvearie/patient-browser/blob/authentication_option_1_takeI/security/README.md

@lmsurpre - still illustrated using direct update to server.xml but haven't forgot about your suggestion of using configDropins.

Also as you'll see, with change to form based authentication required for customised login/error screen, had to set SSO flag to true (so you'll see LTPA tokens now exchanged etc).

### CODE REVIEW

File: build/config/default.json5 [hunk 1]
```diff
@@ -1,3312 +1,2492 @@
...
-            },
-            "271939006": {
-                description: "Leukorrhea",
-                codes: {
-                    "SNOMED-CT": [
-                        "271939006"
-                    ]
-                }
-            },
-            "272036004": {
```
Comment (lmsurpre) [inline build/config/default.json5:2484]: I'm not sure what this file is used for, and its probably just fine, but wanted to ask anyway:  is it ok that we've removed a bunch of codes + descriptions from this list?

File: build/config/default.json5 [hunk 2]
```diff
@@ -1,3312 +1,2492 @@
...
-    },
-    outputMode: "id-list",
-    submitStrategy: "manual"
+  },
+  patientsPerPage: 25,
+  timeout: 20000,
+  renderSelectedOnly: false,
+  fhirViewer: {
+    enabled: true,
+    url: "http://docs.smarthealthit.org/fhir-viewer/index.html",
```
Comment (lmsurpre) [inline build/config/default.json5:2487]: this wasn't a change, but I'm not sure what this url is used for...I wonder if it should get updated to point at localhost?

File: build/config/default.json5 [hunk 3]
```diff
@@ -1,3312 +1,2492 @@
...
-                description: "Primary small cell malignant neoplasm of lung, TNM stage 2 (disorder)",
-                codes: {
-                    "SNOMED-CT": [
-                        "67821000119109"
-                    ]
-                }
-            }
-        }
+  server: {
+    type: "STU-3",
```
[the remark this answers is not in the document]
Comment (mcbride-p) [reply build/config/default.json5:3]: I didn't change type. This is what it was so left it as so. So doesn't seem to matter.

File: build/config/default.json5 (hunk 1 above)
Comment (mcbride-p) [reply build/config/default.json5:2484]: Codes haven't been removed. Is just that there was unusual file formatting in terms of indentation etc previously and standard formatting was introduced when I saved it in VSC.

File: security/README.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
+# How to securely deploy the Patient Browser Application in same container as a FHIR server
+
+## Introduction
+
+Instructions are provided below for a configuration to securely deploy the Patient Browser Application as a servlet in the same container as a FHIR server. THe Patient Browser application will be, an additional protected resource to the FHIR server, deployed on Liberty. Users with correct security role to access the Patient Browser application are configured within the Liberty basic user registry.
```
[the remark this answers is not in the document]
Comment (mcbride-p) [reply security/README.md]: fixed

File: security/README.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+THe base FHIR deployment used is as per helm chart [here](https://github.com/Alvearie/health-patterns/blob/main/helm-charts/fhir/chart/README.md) but modified to include deployment of the Patient Browser application as a protected servlet resource running on the same LIberty server and in the same container.
```
[the remark this answers is not in the document]
Comment (mcbride-p) [reply security/README.md]: Fixed

File: build/config/default.json5 (hunk 2 above)
Comment (mcbride-p) [reply build/config/default.json5:2487]: Doesn't mamake any difference from testing I did, what's in here.

**Pull request merged**